internal/network/containerizer/bridgepolicy.go

```diff
@@ -49,16 +49,19 @@ type BridgePolicy struct {
 	// It's one of:
 	//  - provider
 	//  - local
-	containerNetworkingMethod string
+	containerNetworkingMethod containermanager.NetworkingMethod
 
 	// networkService provides the network domain functionality.
 	networkService NetworkService
 }
 
 // NewBridgePolicy returns a new BridgePolicy for the input environ config
 // getter and state indirection.
-func NewBridgePolicy(ctx context.Context, cfgGetter environs.ConfigGetter, networkService NetworkService) (*BridgePolicy, error) {
-	cfg := cfgGetter.Config()
+func NewBridgePolicy(ctx context.Context,
```

Comment:
One day, this should be pushed behind the service barrier. I believe we'll be able to simplify it at that time.